meshnets/data_processing/torch_utils.py

```diff
@@ -0,0 +1,17 @@
+"""Utility functions for usage with pytorch"""
+import torch
+import torch_geometric
+
+
+def collate_fn(examples,
+               x_key='node_features',
+               edge_index_key='edges',
+               edge_attr_key='edge_features',
+               y_key='wind_pressures'):
+    graphs = [
+        torch_geometric.data.Data(x=torch.tensor(e[x_key]),
```

Comment:
e can mean many things, my first though was edge... Make it for example in examples.